kombu/transport/qpid.py

```diff
@@ -124,6 +124,11 @@
 
 logger = get_logger(__name__)
 
+try:
+    buffer
```

Comment:
You should import buffer from `kombu.five` instead.

Reply:
ImportError: cannot import name 'buffer' from 'vine.five'